packages/admin/cms-admin/src/contentScope/ContentScope.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { ContentScopeInterface } from "./Provider";
+
+export const combineObjects = ([head, ...[headTail, ...tailTail]]: ContentScopeInterface[][]): ContentScopeInterface[] => {
```

Comment:
The name of this util is too generic when passing content scopes as arguments.

Reply:
i have renamed this util to combineContentScopes